pages/contribute/how_to_contribute.md

```diff
@@ -1,39 +1,30 @@
 ---
 title: How to contribute to RDMkit
+toc: false
 ---
 
-## Ways of contributing
+This project is only possible with the many [community contributors](https://rdmkit.elixir-europe.org/contributors). RDMkit is an open project where everyone can contribute to the site.
 
-Everyone is welcome to contribute to this site, and we've tried to make it as easy as possible. Choose a way that suits you. The website is hosted on GitHub, so:
+If you wish to contribute to this community-led project, we ask you to remember the following for better consistency and efficiency: 
 
-* **If you are not familiar with Git** but want to give it a try on GitHub: follow [our step-by-step instructions](github_way) (no technical knowledge required!).
-* **If you are happier using Google Docs**: follow [the Google Doc way](google_doc_way).
-* **If you are familiar with Git**: fork the repo and create a pull request (see our [instructions](working_with_git)).
-* **If you just want to make a quick suggestion**: submit your comments/suggestions using the form below.
+* Follow our [style guide](https://rdmkit.elixir-europe.org/style_guide).
+* Ensure your content respects copyright. Please follow our[ Copyright guidelines](https://rdmkit.elixir-europe.org/copyright).
+* Provide acknowledgement for all your contributions.
 
-## Contributor responsibilities
 
-When writing for this website keep in mind the following:
+## Ways of contributing
 
-* For the sake of consistency, please follow our [style guide](style_guide).
-* It is your responsibility to make sure that the content you provide respects copyright. Please follow our [Copyright guidelines](copyright).
-* Please read our [Markdown cheat sheet](markdown_cheat_sheet) in order to be able to fill in the page metadata and to format the layout correctly.
-* To avoid having to resolve many issues during the revision process, please check the [Editors checklist](editors_checklist) and make sure you have complied with the requirements.
-* If others were involved in your contribution, by writing up or by providing resources such as diagrams or links, please make sure you acknowledge them in the contributors' section of your page, after receiving their permission.
+* Google Doc. If you are unfamiliar with Git and prefer to use a classic document editor, follow [the Google Doc way](https://rdmkit.elixir-europe.org/google_doc_way)
```

Comment:
> ...and prefer to use a classic document editor,...

...and prefer to use a Word-like editor,... ?

Wasn't sure what a "classical" document editor was. QuarkXPress is also a document editor with a long history, but nothing like Google Docs.

Reply:
I agree with docs and started changing changed it further at some places accordingly